_observing-your-data/metricsanalytics.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: default
+title: Metrics analytics
+nav_order: 40
+redirect_from:
+  - /observing-your-data/metricsanalytics/
+---
+
+# Metrics analytics
+Introduced 2.4
+{: .label .label-purple }
+
+With the release of OpenSearch 2.4, you can now ingest and visualize metrics data stored directly within OpenSearch using the **Metrics analytics** tool. This equips you with tools to analyze and correlate data across logs, traces, and metrics.
+
+Before this feature, you could only ingest and visualize logs and traces from your monitored environments. With **Metrics analytics**, you can now observe your digital assets with more granularity, gain deeper insight into the health of your infrastructure, and better inform your root cause analysis.
+
+**Metrics analytics** offers federated visualization capabilities on top of the following:
+
+ - An OpenSearch cluster containing an [OpenTelemetry (OTel)-compatible metrics index](https://github.com/opensearch-project/opensearch-catalog/tree/main/docs/schema/observability/metrics) with OTel-based signals. See [What is OpenTelemetry?](https://opentelemetry.io/docs/what-is-opentelemetry/) for an overview of OTel.
+ - An OpenSearch cluster containing a [Prometheus data source](https://github.com/opensearch-project/sql/blob/main/docs/dev/datasource-prometheus.md) connected to a Prometheus server. 
+
+The following image displays the flow for retrieving metrics from Prometheus and displaying them on a visualization dashboard.
+
+<img src="{{site.url}}{{site.baseurl}}/images/metrics/prom-metrics.png" alt="Prometheus data source" width="700"/>
+
+The following image displays an observability dashboard that visualizes metrics data from the OpenSearch index using OTel queries.
+
+<img src="{{site.url}}{{site.baseurl}}/images/metrics/otel-metrics.png" alt="OTel data source" width="700"/>
+
+---
+
+## Configuring Prometheus to send metrics data to OpenSearch
+
+You must first create a connection from [Prometheus](https://prometheus.io/) to OpenSearch using the [SQL plugin](https://github.com/opensearch-project/sql). You can then configure a connection to Prometheus by using the `_datasources` configuration API endpoint. 
```

Comment:
In "`_datasources` configuration API endpoint", should "Configuration" be capitalized, or can "configuration" be removed?

Reply:
can be removed